lib/components/user/trip-summary.js

```diff
@@ -0,0 +1,76 @@
+import coreUtils from '@opentripplanner/core-utils'
+import { ClassicLegIcon } from '@opentripplanner/icons'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { getQueryParamsFromQueryString } from '../../util/query'
+import ItinerarySummary from '../narrative/default/itinerary-summary'
+
+const { formatDuration, formatTime } = coreUtils.time
+
+class TripSummary extends Component {
```

Comment:
Is this component even needed? I feel like this is a lot of copy-pasted code...

Reply:
As a temporary filler, I have "extracted" this component by copying its counterpart from the default OTP-RR itinerary. Let's coordinate on reusing the trip summary component that is featured in the trip comparison chart.